### DISCUSSION TITLE
Use Website's Own Dark Mode if Available

### DISCUSSION THREAD

**Discussion by webguyio** [darkreader/darkreader#8814] [Ideas]
With the popularity of dark mode having skyrocketed in the past few years, nearly every popular website now supports a dark mode officially.

In most cases, I think it would be preferred to use the official dark mode as opposed to letting Dark Reader try and handle it.

Does Dark Reader already automatically recognize sites that have their own dark mode (Twitter and Reddit for example) and attempt to toggle that on or insert a cookie declaring that we prefer dark mode? If not, I guess I'm suggesting the idea.

`prefers-color-scheme` does a great deal to solve this problem already for many, only depending on your browser's privacy settings, it may not.

**Comment by Gusted** [comment]
See https://github.com/darkreader/darkreader/issues/1285

**Comment by webguyio** [comment]
I just finished skimming that entire issue, and what a ride that was!

Ultimately, it seems like a very difficult issue with no easy solution, but if you think there's another takeaway I should have, please give me a different summary (if you don't mind). Thanks.
